frontend/src/journal/pages/Journals.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import remove from "../../assets/trash.svg";
+import useGetJournals from "../hooks/useJournal";
+import { formatDate } from "../utils/formatDate";
+
+const JournalsList: React.FC = () => {
+  const {
+    loading,
+    error,
+    data,
+    handleDeleteJournal,
+    handleSelectJournal,
+    selectedId,
+  } = useGetJournals();
+
+  if (loading) return <p>Loading journals...</p>;
+  if (error) return <p>Error loading journals: {error.message}</p>;
+
+  const journals = data?.getJournals
```

Comment:
I would move this logic to useGetJournals hook. And instead of returning `data`, return `journals` instead

A good practice, is to move that kind of logic away from the UI layer, to keep it simpler

Reply:
Done!